apis/v1alpha2/validation/httproute.go

```diff
@@ -107,8 +108,8 @@ func validateHTTPRouteFilters(filters []gatewayv1a2.HTTPRouteFilter, matches []g
 		delete(counts, key)
 	}
 
-	if counts[gatewayv1a2.HTTPRouteFilterRequestRedirect] > 0 && counts[gatewayv1a2.HTTPRouteFilterURLRewrite] > 0 {
-		errs = append(errs, field.Invalid(path.Child("filters"), gatewayv1a2.HTTPRouteFilterRequestRedirect, "Redirect and Rewrite filters cannot be defined in the same list of filters"))
+	if counts[gatewayv1b1.HTTPRouteFilterRequestRedirect] > 0 && counts[gatewayv1b1.HTTPRouteFilterURLRewrite] > 0 {
+		errs = append(errs, field.Invalid(path.Child("filters"), gatewayv1b1.HTTPRouteFilterRequestRedirect, "Redirect and Rewrite filters cannot be defined in the same list of filters"))
```

Comment:
Given other examples in kube core and networking, perhaps this language would be better?

```suggestion
		errs = append(errs, field.Invalid(path.Child("filters"), gatewayv1b1.HTTPRouteFilterRequestRedirect, "may specify httpRouteFilterRequestRedirect or httpRouteFilterRequestRewrite, but not both"))
```

Reply:
Updated, thanks!